scripts/build_shapes.py

```diff
@@ -858,7 +857,13 @@ def gadm(
             name_file_nc="GDP_PPP_1990_2015_5arcmin_v2.nc",
         )
 
-    # set index and simplify polygons
+    # renaming 3 letter to 2 letter ISO code before saving GADM file
+    # solves issue: https://github.com/pypsa-meets-earth/pypsa-earth/issues/671
+    df_gadm["GADM_ID"] = (
+        df_gadm["GADM_ID"]
+        .str.split(".")
+        .apply(lambda id: three_2_two_digits_country(id[0]) + "." + id[1])
```

Comment:
Should be this instead:
`.apply(lambda id: three_2_two_digits_country(id[0]) + "." + id[1:])`